src/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import TodoListItem from './TodoListItem';
+
+/////////////////////////////
+// The todo arry
+////////////////////////////
+
+const todoList = [
+     {
+       title: 'HelloReact',
+      id: 1,
+   }, {
+       title: 'Requirements',
+       id: 2,
+   },{
+     title: 'SettingupaReactProject',
+     id: 3,
+   },{
+     title: 'MeettheReactComponent.',
+     id: 4,
+   },{
+     title: 'React JSX',
+     id: 5,
+   },{
+     title: 'Lists in React',
+     id: 6,
+   },
+   
+   
+   ];
+
+////////////////////////////////   
+// The todo function
+///////////////////////////////
+
+function TodoList () {
+    return (
+        <ul>
+        {/*This map function will iterate through the array of object*/}
+      
+      {todoList.map((todo) =><TodoListItem key={todo.id}  title={todo.title}  /> )} 
+      
+    
+      </ul>
+    )
+}
+      
+   
+    
+
+  
+
+export default TodoList;
```

Comment:
This block of code is functionally fine, but difficult to read due to all of the extra whitespace and non-uniform indentation. I'll explain further in a DM, but you should really consider installing eslint and prettier. Below is how this component should look when properly formatted. 

```suggestion
function TodoList() {
  return (
    <ul>
      {/*This map function will iterate through the array of object*/}

      {todoList.map((todo) => (
        <TodoListItem key={todo.id} title={todo.title} />
      ))}
    </ul>
  );
}

export default TodoList;
```